### PR TITLE
Add an boolean property to clear the OTP Input fields

### PR DESCRIPTION
**Pull request by hasithaWalpola**

### CODE REVIEW

**Comment by fpena** [comment]
Hey @hasithaWalpola. Thanks for the contribution. 
I just merged another PR. You might want to rebase/merge `master` and update this one. 
Thanks and let me know if you have any questions!

**Comment by hasithaWalpola** [comment]
Hi, @fpena. I resolved the conflicts and committed. Can you check that.

File: index.js [hunk 1]
```diff
@@ -188,12 +192,12 @@ export default class OTPInputView extends Component {
...
-                    value={digits[index]}
-                    keyboardType={keyboardType}
-                    textContentType={isAutoFillSupported ? "oneTimeCode" : "none"}
+                    value={!clearInputs ? digits[index]: ""}
+                    keyboardType="number-pad"
+                    textContentType= {isAutoFillSupported ? "oneTimeCode" : "none"}
+
                     key={index}
                     selectionColor="#00000000"
-                    secureTextEntry={secureTextEntry}
```
Comment (fpena) [inline index.js]: @hasithaWalpola why are we deleting this?

Comment (hasithaWalpola) [reply index.js]: It was deleted mistakenly. It should be there. Sorry for that

File: index.js [hunk 2]
```diff
@@ -38,13 +40,6 @@ export default class OTPInputView extends Component {
         }
     }
 
-    componentWillReceiveProps(nextProps) {
```
Comment (fpena) [inline index.js]: @hasithaWalpola what about this code? I think it should remain in the library.

Comment (hasithaWalpola) [reply index.js]: yes, it should remain

**Pull request merged**